persistence-cassandra/javadsl/src/test/scala/com/lightbend/lagom/javadsl/persistence/cassandra/CassandraReadSideSpec.scala

```diff
@@ -17,15 +20,35 @@ import com.lightbend.lagom.javadsl.persistence.Offset.TimeBasedUUID
 import com.lightbend.lagom.javadsl.persistence._
 import com.typesafe.config.ConfigFactory
 import play.api.inject.guice.GuiceInjectorBuilder
-
 import scala.compat.java8.FutureConverters
 import scala.concurrent.Await
 import scala.concurrent.Future
 import scala.concurrent.duration._
 
 object CassandraReadSideSpec {
-  val defaultConfig      = ConfigFactory.parseString("akka.loglevel = INFO")
-  val noAutoCreateConfig = ConfigFactory.parseString("lagom.persistence.read-side.cassandra.tables-autocreate = false")
+  def firstTimeBucket: String = {
+    val today                                = LocalDateTime.now(ZoneOffset.UTC)
+    val firstBucketFormat: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMdd'T'HH:mm")
+    today.minusHours(3).format(firstBucketFormat)
```

Comment:
What mean these 3 hours? 🤔 Just recent past?

Reply:
Yeah. A recent past will make the read-side processors to start faster when there is no data in the journal.

Reply:
@ignasi35 👌